packages/suite-desktop/src-electron/electron.ts

```diff
@@ -75,7 +76,7 @@ const init = async () => {
         // do nothing
     }
 
-    if (isDev) {
+    if (isDev && !(await isPortUsed(8000))) {
```

Comment:
This was added to check if the Next dev server is running. We use `electron-next` which only provides a method to run the Next dev server but none for shutting it down or checking if it's up. Because the app is not killed on Mac when you close it, the server stays alive.

Reply:
I changed the code to use a boolean to check if `init` was already run or not. I kept the port checking code function in the utils file for some future work.